src/checkers/periodicAwsChecker.go

```diff
@@ -43,7 +43,8 @@ func (p *PeriodicAwsChecker) StartChecking() {
 		p.exporter.ResetMetrics()
 
 		// Create a Session with a custom region
-		svc := secretsmanager.New(session.New(), aws.NewConfig().WithRegion(p.awsRegion))
+		session, _ := session.NewSession()
```

Comment:
if this returns an error we should log it and increment the errors metric

Reply:
Hey @joe-elliott. Definitely makes sense. :) Added logging and error counter increment. Thanks for the swift reply!